### PR TITLE
Remove our replacement for Logrus

### PR DESCRIPTION
**Pull request by jentfoo**
Recently I attempted to update our Logrus fork.  However this comment pointed out that our changes have been merged upstream: https://github.com/gravitational/logrus/pull/12#issuecomment-1515303744

For that reason this PR removes the dependency on the fork.

### CODE REVIEW

**Comment by tobiaszheller** [comment]
can you add release notes from v1.4.4 to v1.9.0 to this PR?

**Comment by jentfoo** [comment]
Logrus highlights the following changes:
```
1.5.0:
    Ability to DisableHTMLEscape when using the JSON formatter: https://github.com/sirupsen/logrus/pull/524
    Support/fixes for go 1.14
    Many many bugfixes

1.7.0:
    a new buffer pool management API has been added
    a set of <LogLevel>Fn() functions have been added
    the dependency toward a windows terminal library has been removed

1.7.1:
Code quality:
    use go 1.15 in travis
    use magefile as task runner
Fixes:
    small fixes about new go 1.13 error formatting system
    Fix for long time race condiction with mutating data hooks
Features:
    build support for zos

1.8.0:
    Correct versioning number replacing v1.7.1

1.8.1:
    Update changelog: https://github.com/sirupsen/logrus/pull/1244

1.9.0:
  Update dependencies: https://github.com/sirupsen/logrus/pull/1343
```

File: go.mod [hunk 1]
```diff
@@ -120,7 +120,7 @@ require (
 	github.com/russellhaering/goxmldsig v1.4.0
 	github.com/schollz/progressbar/v3 v3.13.1
 	github.com/sethvargo/go-diceware v0.3.0
-	github.com/sirupsen/logrus v1.9.0 // replaced
+	github.com/sirupsen/logrus v1.9.0
```
Comment (codingllama) [inline go.mod:123]: Please edit .github/dependabot.yml too.

**Review by codingllama — APPROVED** [review]
Nice.

File: go.mod (hunk 1 above)
Comment (jentfoo) [reply go.mod:123]: Thank you, updated.  I also removed `/pkg/sftp` which is no longer replaced as of 40c113b8dad017613c154efaa16997a186eec7fb

**Review by tobiaszheller — APPROVED** [review]

**Review by zmb3 — APPROVED** [review]
another one bites the dust

**Pull request merged**